### ISSUE TITLE
Better distinguish "Verified feed availability" and "Verified feed directory"

### ISSUE THREAD

**Issue by MikeThacker1** [tpximpact/mhclg-oruk#48]
[Verified feed availability](https://openreferraluk.netlify.app/developers/dashboard) and [Verified feed directory](https://openreferraluk.netlify.app/community/directory) are two different things, which was not clear to me until recently and will need to be learned buy people who were used to the old single [Dashboard](https://openreferraluk.org/dashboard).

We need the UI to make this difference very clear and, I think, we need separate unambiguous names for these two things.

**Comment by gjhltn** [comment]
This was Laura's preferred wording. I am neither wedded to it nor do I have a problem with it. The two are views of the same thing, so there are arguments for and against. 

That said, I don't think that learnabilty is an issue either way. If you used the dashboard, its new name is VFA... the naming of VFD doesn't affect that.

**Comment by MikeThacker1** [comment]
Of course I bow to @LauraTPX's decision, particularly if @gjhltn thinks it makes sense. It's just a casse of me learning the new naming and converting existing users to it.

Note that  the [Technical overview to implementing Open Referral UK](https://openreferraluk-development.netlify.app/developers/overview) refers in section 5.2 to [verified feed dashboard](https://openreferraluk-development.netlify.app/developers/dashboard). We might want to change that wording although the page URL may not need to be changed.

**Comment by MikeThacker1** [comment]
> Note that the [Technical overview to implementing Open Referral UK](https://openreferraluk-development.netlify.app/developers/overview) refers in section 5.2 to [verified feed dashboard](https://openreferraluk-development.netlify.app/developers/dashboard). We might want to change that wording although the page URL may not need to be changed.

I've made the change from "verified feed dashboard" to "verified feed availability page" in both" section 5.2 of [Technical overview to implementing Open Referral UK](https://openreferraluk-development.netlify.app/developers/overview) and  [section 5.2 of the Google document ORUK Technical overview](https://docs.google.com/document/d/1XU9aWCn37jjh6mP51VsRADImUXB8R8Jgq1IqbUEfSBk/edit?tab=t.0#heading=h.w4ad74nk0zkk).

**Comment by MikeThacker1** [comment]
Agreed with @LauraTPX that we leave the wording and explanations of [Verified feed availability](https://openreferraluk.netlify.app/developers/dashboard) and [Verified feed directory](https://openreferraluk.netlify.app/community/directory) as they are for now,

**These should be reviewed in the light of live user feedback.**

**Comment by gjhltn** [comment]
Ok. so closing this for no because no action needed right now
